data/.gitignore

```diff
@@ -0,0 +1 @@
+*
```

Comment:
I think the convention is to call these `.gitkeep` so that the folder can be tracked, then ignoring it in `.gitignore`

Reply:
The aim is not only to have this dir in git but also have it ignore any files you actually copy under it (i.e. datasets)